roles/openshift_logging/defaults/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+openshift_logging_curator_image_prefix: "{{ openshift_logging_image_prefix | default(__openshift_logging_image_prefix) }}"
```

Comment:
will `__openshift_logging_image_prefix` be set correctly here? I think we may need to load it in in the main task instead